packages/cursorless-engine/package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "compile": "tsc --build",
     "watch": "tsc --build --watch",
-    "clean": "rm -rf ./out tsconfig.tsbuildinfo"
+    "clean": "rm -rf ./out tsconfig.tsbuildinfo",
+    "generate-test-subset-file": "tsx --conditions=cursorless:bundler ./src/scripts/generateTestSubsetFile/generateTestSubsetFile.ts",
```

Comment:
Looks like you dropped the bundle arg we always use with `tsx`. That will cause unpredictable behaviour if you import something from another package

See
https://youtu.be/Od7BZHc7w6c for more than you probably ever wanted to know on the subject 😄

Reply:
thanks! will watch that video after the meetup